### PR TITLE
Fix duplication in scheduler.

### PR DESCRIPTION
**Pull request by aveshagarwal**
**What this PR does / why we need it**:
This fixes duplicate code in scheduler.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
No issue.
**Special notes for your reviewer**:

**Release note**:

```release-note
```
@kubernetes/sig-scheduling-pr-reviews @bsalamat

### CODE REVIEW

**Comment by bsalamat** [comment]
Hmm...This piece of code that this PR removes is added recently by @caesarxuchao in https://github.com/kubernetes/kubernetes/pull/39173.

Chao,
Could you please chime in an let us know why that change was needed?

File: plugin/cmd/kube-scheduler/app/configurator.go [hunk 1]
```diff
@@ -58,10 +57,10 @@ func createRecorder(kubecli *clientset.Clientset, s *options.SchedulerServer) re
 }
 
 // TODO: convert scheduler to only use client-go's clientset.
```
Comment (caesarxuchao) [inline plugin/cmd/kube-scheduler/app/configurator.go]: Remove the TODO please.

**Comment by caesarxuchao** [comment]
A nit. Otherwise lgtm. Thanks @aveshagarwal.

@bsalamat the code was in an intermediate state.

**Comment by bsalamat** [comment]
@caesarxuchao Thanks for clarification!

**Comment by aveshagarwal** [comment]
@bsalamat @caesarxuchao done. thanks.

**Comment by bsalamat** [comment]
/lgtm

@aveshagarwal Thanks for the cleanup.

**Comment by k82cn** [comment]
/approve no-issue

Thanks for the cleanup :).

**Pull request merged**